comp413-2017/RDFS
# Fix ReplicationTest

## Overview
Fixes ReplicationTest, which checks to make sure that file system data is being replicated across multiple DataNodes.

## Testing
1. `git checkout feature/replication_test`
2. Go to the `build/test` directory and `make`
3. Run `./ReplicationTest`
4. See that 2 tests pass: `testReadWrite` and `testReplication`
